src/index.ts

```diff
@@ -194,6 +194,11 @@ export async function generateDiamondAbi(
       log(`Skipping ${contractName} because it did matched an \`exclude\` pattern.`);
       continue;
     }
+    
+    // this should be the output filename, but this will work too
+    if (contractName.match("hardhat-diamond-abi\/.*")) {
```

Comment:
```suggestion
    if (contractName.startsWith(PLUGIN_NAME)) {
```
Wondering if it should be like this, which is more resilient to changing the plugin name and doesn't do another match (I believe startsWith is faster.

I also think I want to write a test for this, and log that it was skipped.